cooltools/api/expected.py

```diff
@@ -938,13 +961,44 @@ def expected_cis(
     map_functor : callable, optional
         Map function to dispatch the matrix chunks to workers.
         If left unspecified, pool_decorator applies the following defaults: if nproc>1 this defaults to multiprocess.Pool;
-        If nproc=1 this defaults the builtin map. 
+        If nproc=1 this defaults the builtin map.
 
     Returns
     -------
     DataFrame with summary statistic of every diagonal of every symmetric
     or asymmetric block:
-    region1, region2, diag, n_valid, count.sum count.avg, etc
+
+    Notes
+    -----
+    Output DataFrame columns correspond to the following quantities:
```

Comment:
do you think this docstring would be clearer if grouped into the default, and then what would be returned if clr_weight_name=None ?

Reply:
I separated them into four cases